rh_region.go

```diff
@@ -0,0 +1,61 @@
+package sdk
+
+import (
+	json2 "encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+type Region struct {
+	URL string
+	AWS []string
+	GCP []string
+}
+
+func GetRhRegions(ocmServiceUrl string) (map[string]Region, error) {
+	var regions map[string]Region
+	url, err := DetermineRegionDiscoveryUrl(ocmServiceUrl)
+	if err != nil {
+		return regions, fmt.Errorf("Can't determine region discovery URL: %s\n", err)
+	}
+	// Adding nolint here in order to prevent linter from failing due to variable http get
+	resp, err := http.Get(url) //nolint
+	if err != nil {
+		return regions, fmt.Errorf("Can't retrieve shards: %s", err)
+	}
+	err = json2.NewDecoder(resp.Body).Decode(&regions)
+	if err != nil {
+		return regions, fmt.Errorf("Can't decode shards: %s", err)
+	}
+	return regions, nil
+}
+
+func GetRhRegion(ocmServiceUrl string, regionName string) (Region, error) {
+	regions, err := GetRhRegions(ocmServiceUrl)
+	if err != nil {
+		return Region{}, err
+	}
+	for regName, regValue := range regions {
+		if regName == regionName {
+			return regValue, nil
+		}
+	}
+	return Region{}, fmt.Errorf("Can't find region %s", regionName)
+}
+
+func DetermineRegionDiscoveryUrl(ocmServiceUrl string) (string, error) {
+	baseUrl, err := url.Parse(ocmServiceUrl)
+	if err != nil {
+		return "", err
+	}
+	regionDiscoveryHost := "api.openshift.com"
+	//TODO: Remove the OR condition from this if statement before the MR merge
+	if strings.HasSuffix(baseUrl.Hostname(), "integration.openshift.com") || true {
```

Comment:
get rid of the //TODO comment and the `|| true` and i think this MR will be good to go

Reply:
Removed TODO comment and always true condition